roles/nova-data/tasks/libvirt.yml

```diff
@@ -57,28 +57,34 @@
   when: openstack_install_method == 'source'
 
 - name: update various lines in libvirtd.conf
-  lineinfile: dest=/etc/libvirt/libvirtd.conf regexp="{{ item.value.regexp }}"
-              line="{{ item.value.line }}"
+  lineinfile:
+    dest: /etc/libvirt/libvirtd.conf
+    regexp: "{{ item.value.regexp }}"
+    line: "{{ item.value.line }}"
   with_dict:
-    ltcp:
+    listen_tcp:
       regexp: '^listen_tcp\s*='
       line: "listen_tcp = 1"
-    ltls:
+    listen_tls:
       regexp: '^listen_tls\s*='
       line: "listen_tls = 0"
-    atcp:
+    auth_tcp:
       regexp: '^auth_tcp\s*='
       line: 'auth_tcp = \"none\"'
   notify: restart libvirt-bin
 
 - name: update libvirtd default
-  lineinfile: dest=/etc/default/libvirt-bin regexp="^libvirtd_opts\s*="
-              line="libvirtd_opts=\"-d -l\""
+  lineinfile:
+    dest: /etc/default/libvirt-bin
+    regexp: '^libvirtd_opts\s*='
+    line: 'libvirtd_opts=\"-d -l\"'
```

Comment:
Lets also take out the -d -l bits here, maybe? Well, once you get it to work, we can try taking it out.